### PR TITLE
msgpack_c 6.0.0 -> 6.0.1

### PR DESCRIPTION
**Pull request by Zopolis4**
No filelist updates.

### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/Zopolis4/chromebrew.git CREW_BRANCH=slant crew update
```

### CODE REVIEW

File: packages/msgpack_c.rb [hunk 1]
```diff
@@ -1,36 +1,26 @@
...
-       i686: '23f44da7a26b62bc6b38be11b7c2a5cefc1b2ac11a5d87982f73a4a2edb8ad03',
-     x86_64: '4171f5ad1d9e9b4e40b06e0e9f68173edace4f25d7d95e8b1f9e581e45a87f7d'
+    aarch64: '06550bcf508a6afe4e7179194b537f967f4577cb7c89f66c9cfd3baa579d6074',
+     armv7l: '06550bcf508a6afe4e7179194b537f967f4577cb7c89f66c9cfd3baa579d6074',
+       i686: '86fe5469cd32609648ac32bb6a2347f730a00ae1e4adaff8c8861aeb99859327',
+     x86_64: 'f4c07ff6634eaf48926ade547924086349bbdc8218f69038b7be219a9c544c11'
   })
 
   depends_on 'doxygen' => :build
-  depends_on 'glibc' # R
```
Comment (satmandu) [inline packages/msgpack_c.rb:24]: It doesn't depend upon glibc? What does `getrealdeps.rb` say?

File: packages/msgpack_c.rb [hunk 2]
```diff
@@ -1,36 +1,27 @@
...
   depends_on 'doxygen' => :build
   depends_on 'glibc' # R
+  depends_on 'gtest' => :build
 
-  def self.patch
-    system "sed -i 's,exec_prefix}/lib,exec_prefix}/#{ARCH_LIB},g' CMakeLists.txt"
-    system "sed -i 's,CMAKE_INSTALL_PREFIX}/lib,CMAKE_INSTALL_PREFIX}/#{ARCH_LIB},g' CMakeLists.txt"
-    system "sed -i 's,CMAKE_INSTALL_LIBDIR lib,CMAKE_INSTALL_LIBDIR #{ARCH_LIB},g' CMakeLists.txt"
-  end
+  cmake_options '-DBUILD_SHARED_LIBS=ON -DMSGPACK_BUILD_TESTS=ON -DMSGPACK_BUILD_EXAMPLES=OFF'
```
Comment (satmandu) [inline packages/msgpack_c.rb:24]: Having `BUILD_TESTS=ON` is fine, but then do we need `run_tests` set in the package, or is that the default now?

Comment (Zopolis4) [reply packages/msgpack_c.rb:24]: It is not the default.

Comment (satmandu) [reply packages/msgpack_c.rb:24]: Do you want to add `run_tests` then?

Comment (satmandu) [reply packages/msgpack_c.rb:24]: Otherwise why build tests?

Comment (Zopolis4) [reply packages/msgpack_c.rb:24]: I did add `run_tests`.

**Review by satmandu — APPROVED** [review]

**Pull request merged**